### ISSUE TITLE
Set field value a a variable depending on another fields value

### ISSUE THREAD

**Issue by laahs** [rs/rest-layer#20]
For example on a schema I could have fields such as "city"=city and "business-name"=the_name. And use those to build a url friendly id and access the page via domain.com/business/the_name_city.

What would be the easiest way to do it?

I don't really understand how to access the data in it's "storable" version once compiled from the schema ...

**Comment by rs** [comment]
At resource level you can bind the Insert and Update events to compute this field.

**Comment by laahs** [comment]
Ok for example i should use inserteventhandlerfunc which has an oninsert method taking a context and a payload (map[string]interface) and here the payload is the map of field names/values of my resources, defined via the schema. Right?

So I can access their "final" values and manipulate them before sending to the storer. Is that the procedure ? (Sorry to bother you but I'm on my phone so i can not try the procedure instead of asking you to confirm that this is what you meant)

**Comment by rs** [comment]
That's exactly the idea yes.

**Comment by laahs** [comment]
Okay perfect, I'll try it. I was missing the part where I have access to the final representation of the data, as transfered to the storer ... therefor was I locked in the "schema" phase with init/update functions that can be attached to the field, but fields values are isolated in the schema ... this was where I got stuck. Thank you for your response :) 

Have you delivered a ffew projects using rest-layer yet?
